src/validation.js

```diff
@@ -0,0 +1,28 @@
+const supportedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const errorMessages = {
+  missingText: 'Text to convert is required. '
+  + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  missingCase: '"toCase" query param is required. '
+  + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  notValidCase: 'This case is not supported. '
+  + 'Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+};
+
+function validation(textToConvert, caseName) {
+  const errors = [];
+
+  if (!textToConvert) {
+    errors.push({ message: errorMessages.missingText });
+  }
+
+  if (!caseName) {
+    errors.push({ message: errorMessages.missingCase });
+  } else if (!supportedCases.includes(caseName)) {
+    errors.push({ message: errorMessages.notValidCase });
+  };
```

Comment:
Let's avoid `else` and `else if` construction

```suggestion
  if (!caseName) {
    errors.push({ message: errorMessages.missingCase });
  }
  
  if (caseName && !supportedCases.includes(caseName)) {
    errors.push({ message: errorMessages.notValidCase });
  };
```